README.md

```diff
@@ -121,7 +140,7 @@ Taking the above into account, the following combinations for `install.yaml` are
        source:
          type: system
          name: <Package name>
-     indigo:
+     noetic:
```

Comment:
Why did you flip it back from the default?

Now the example is not clear that `default` can also have something other than `pass`.

Reply:
That was not clear from your changes.

Reply:
I tried to catch that in the description:
> and show default can also have a source.

But failed I guess :smile:

Reply:
Haha, 'read the manual' == 'read the commit message'

Reply:
Well descriptive commit messages are important!